### DISCUSSION TITLE
Add unit to "Number" field

### DISCUSSION THREAD

**Discussion by iero** [twentyhq/twenty#3954] [Ideas]
For example to setup a "mm" for a measure or "ha" for a surface

**Comment by Bonapara** [comment]
Hi @iero, thank you for sharing your idea! I'm curious, what is your specific use case? Have you considered using a column labeled "XXX in mm" or "XXX in ha" to do the job?

**Comment by iero** [comment]
Thanks @Bonapara for the feedback!

Yes the goal is to print the unit in the frontend to be sure that we are not entering acres or km2 when we have to put hectare for example. After this information can be used with the API to convert data before pushing it to the database, or throw an alert if the units are not the same.

I can put it in the label and regex the label for sure but is it the cleaner way to handle this?

**Comment by FelixMalfait** [comment]
Thanks for the suggestion! We're going to introduce a composite field like this for currencies. A composite fields is just 2 separate fields in the backend. So you could create 2 fields: a select field for units + a number field, or as Thomas mentioned you could put the unit in the field name. I will close the discussion since it's unlikely we implement anything more short-term but feel free to reply.
